packages/frontend/src/utils/queries.ts

```diff
@@ -5,6 +5,10 @@ export function getUrl(chain) {
   if (chain === 'ethereum') {
     chain = 'mainnet'
   }
+
+  if (chain === 'optimism') {
+    chain = 'optimism1'
```

Comment:
we can actually use `'hop-optimism` again now (no need for `hop-optimism1` anymore)